tofhir-server/src/main/scala/io/tofhir/server/service/job/IJobRepository.scala

```diff
@@ -52,4 +52,12 @@ trait IJobRepository {
    * @return
    */
   def deleteJob(projectId: String, id: String): Future[Unit]
+
+  /**
+   * Retrieves the jobs referencing the given mapping in their definitions.
+   * @param projectId identifier of project whose jobs will be checked
+   * @param mappingUrl the url of mapping
+   * @return the jobs referencing the given mapping in their definitions
+   */
+  def getJobsReferencingMapping(projectId: String, mappingUrl: String): Future[Seq[FhirMappingJob]]
```

Comment:
I am not sure whether these methods should be part of the API. We would not have needed it if had a relational db based implementation. But, let's keep them for now